public/build/sylius/orderCreation/shop/manifest.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
1. Built assets shouldn't be included in git
2. public/build shouldn't exists in plugin unless it's a test app